README.md

```diff
@@ -32,14 +32,14 @@ exists to maintain and develop:
 
 ## Roles
 
-### Core Owners
+### Core Team Members
 
-The core organization owners lead the development of the language, compiler,
+The core organization team members lead the development of the language, compiler,
```

Comment:
I'd shorten this to "The core team lead the development..."

Reply:
Done in latest commits.